subprojects/reporting/src/main/java/org/gradle/api/reporting/ReportingExtension.java

```diff
@@ -143,5 +143,5 @@ public String getApiDocTitle() {
     @Incubating
     public ExtensiblePolymorphicDomainObjectContainer<ReportSpec> getReports() {
         return reports;
-    }
+    } //todo: potentially fix the same way as TestingExtension.getSuites()
```

Comment:
Good catch. This can happen in another PR though

Reply:
Did it here, was a simple change.